src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -241,13 +255,21 @@ class MockClientBackend : public ClientBackend {
     return Error::Success;
   }
 
-  Error ClientInferStat(InferStat* a) override { return Error::Success; }
+  Error ClientInferStat(InferStat* a) override
```

Comment:
nit: can we have a better parameter name here?

Reply:
fixed